feature/bookmarks/src/main/java/com/google/samples/apps/nowinandroid/feature/bookmarks/BookmarksScreen.kt

```diff
@@ -74,18 +71,6 @@ fun BookmarksScreen(
     modifier: Modifier = Modifier
 ) {
     Scaffold(
```

Comment:
Nit: Once we remove the `topBar` here, it looks like the `Scaffold` doesn't do anything?

Reply:
I removed it and couldn't see any changes to the layout so 👍.